mobsfscan/rules/patterns/ios/swift/best_practices.yaml

```diff
@@ -78,6 +78,8 @@
     - \.autocorrectionType\s*=\s*\.No
     - UITextAutocorrectionTypeNo
     - \.disableAutocorrection\s*\(\s*true\s*\)
+    - \.autocorrectionDisabled\s*\(\s*true\s*\)
+    - \.autocorrectionDisabled\s*\(\)
```

Comment:
Can we remove this? Let's not flag on the ones we cannot explicitly detect as `true`

Reply:
I removed that last line with default parameter